suppression_list.go

```diff
@@ -136,9 +179,9 @@ func (c *Client) SuppressionUpsertContext(ctx context.Context, entries []Suppres
 	}
 
 	path := fmt.Sprintf(SuppressionListsPathFormat, c.Config.ApiVersion)
-	list := SuppressionListWrapper{nil, entries, 0, nil}
+	suppressionPage := SuppressionPage{}
 
-	jsonBytes, err := json.Marshal(list)
+	jsonBytes, err := json.Marshal(suppressionPage)
```

Comment:
This error check can be removed - the exported fields on `SuppressionPage` can't make `Marshal` fail. It would have to contain an `interface{}`, or some other custom type that isn't serializable.